internal/events/token_pool_created.go

```diff
@@ -165,7 +165,7 @@ func (em *eventManager) TokenPoolCreated(ti tokens.Plugin, pool *tokens.TokenPoo
 		// Initiate a rewind if a batch was potentially completed by the arrival of this transaction
 		if batchID != nil {
 			log.L(em.ctx).Infof("Batch '%s' contains reference to received pool '%s'", batchID, pool.ProtocolID)
-			em.aggregator.offchainBatches <- batchID
+			em.aggregator.rewindBatches <- batchID
```

Comment:
This is an example of why I left the rename - as the `offchain` felt a bit odd here.